autotest/ogr/ogr_mssqlspatial.py

```diff
@@ -569,6 +606,7 @@ def test_ogr_mssqlspatial_cleanup():
     gdaltest.mssqlspatial_ds = ogr.Open(gdaltest.mssqlspatial_dsname, update=1)
     gdaltest.mssqlspatial_ds.ExecuteSQL("DROP TABLE Unregistered")
     gdaltest.mssqlspatial_ds.ExecuteSQL("DROP TABLE tpoly")
+    gdaltest.mssqlspatial_ds.ExecuteSQL("DROP TABLE poly")
```

Comment:
IMHO the practice of the test_XXXX_cleanup() method should be considered as deprecated. Each test step should clean up after itself so they can be run in a isolated way.
So concretely I would put lines 580-592 inside a try block and put ``gdaltest.mssqlspatial_ds.ExecuteSQL("DROP TABLE poly")`` in the finally clause